sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/HttpTimeoutPolicyControlPlaneHotPath.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.http;
+
+import io.netty.handler.codec.http.HttpMethod;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class HttpTimeoutPolicyControlPlaneHotPath extends HttpTimeoutPolicy {
+
+    public static final HttpTimeoutPolicy INSTANCE = new HttpTimeoutPolicyControlPlaneHotPath();
+
+    private HttpTimeoutPolicyControlPlaneHotPath() {
+    }
+
+    @Override
+    public List<ResponseTimeoutAndDelays> getTimeoutList() {
+        return Collections.unmodifiableList(Arrays.asList(new ResponseTimeoutAndDelays(Duration.ofMillis(500), 0),
```

Comment:
since this list will never change, so probably can just declare in the constructor or during static initialization, instead of creating one everytime

Reply:
As discussed previously, please consider using a static list instead initializing a new one each time